server.js

```diff
@@ -95,5 +96,9 @@ app.post('/unwrap', function(req, res) {
 app.get('/');
 
 app.get('*', function (req, res) {
-    res.render(path.join(__dirname, '/index.html'),{defaultUrl: VAULT_URL_DEFAULT, defaultAuth: VAULT_AUTH_DEFAULT});
+    res.render(path.join(__dirname, '/index.html'),{
+        defaultUrl: VAULT_URL_DEFAULT,
+        defaultAuth: VAULT_AUTH_DEFAULT,
+        suppliedAuthToken: VAULT_SUPPLIED_TOKEN_HEADER ? req.header(VAULT_SUPPLIED_TOKEN_HEADER) : ""
```

Comment:
This is actually needed, otherwise in situations where the env var is not set it will result in a `req.header("")` call which returns the following exception:
```
TypeError: name argument is required to req.get
   at IncomingMessage.header (/app/node_modules/express/lib/request.js:61:11)
   at /app/server.js:102:32
   at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
   at next (/app/node_modules/express/lib/router/route.js:131:13)
   at Route.dispatch (/app/node_modules/express/lib/router/route.js:112:3)
   at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)
   at /app/node_modules/express/lib/router/index.js:277:22
   at param (/app/node_modules/express/lib/router/index.js:349:14)
   at param (/app/node_modules/express/lib/router/index.js:365:14)
   at Function.process_params (/app/node_modules/express/lib/router/index.js:410:3)
```